data-processing/entities/sentences_pdf/__init__.py

```diff
@@ -0,0 +1,30 @@
+from common.commands.base import CommandList
```

Comment:
So just to be clear that I understand -- sentence processing is represented at the meta level by this command list. We are explicitly only using SPP for sentence bbox detection at this time, right?

Separately, is this command list loaded if you ask for `entities="*"`, or only if you ask for `entities="sentences"`?